App.News/Controllers/NewsController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using App.News.Models;
+using App.News.Interfaces;
+
+namespace App.News.Controllers
+{
+    [Route("api/news")]
+    [ApiController]
+    public class NewsController:Controller
+    {
+        INewsManager newsManager;
+
+        public NewsController(INewsManager newsManager)
+        {
+            this.newsManager = newsManager;
+        }
+        [HttpPost]
+        public ActionResult AddComment([FromBody]CommentDTO commentDTO)
+        {
+            var actionResult = newsManager.AddComment(commentDTO);
+            if (actionResult != "ok")
+            {
+                return new BadRequestResult();
+            }
+            return new OkResult();
+        }
+        [Route("api/news/allNews")]
+        [HttpGet]
```

Comment:
1. U can write it in one line -  [HttpGet("your/route")]
2. You don't need add "api" part in this rout, because you already have word "api" before controller declaration. In you case request url for this rout will be like "api/news/api/news/allNews"
3. Correct route must look like [HttpGet("all")] or somthing like this